packages/financial-templates-lib/src/clients/ExpiringMultiPartyClient.js

```diff
@@ -38,12 +51,12 @@ class ExpiringMultiPartyClient {
     this.toBN = this.web3.utils.toBN;
     this.toWei = this.web3.utils.toWei;
 
-    const Convert = (decimals = 18) => number => this.toBN(parseFixed(number.toString(), decimals).toString());
-
-    // currently not implemented
-    this.convertSynthetic = Convert(syntheticDecimals);
-    this.convertCollateral = Convert(collateralDecimals);
-    this.convertCollateralToSynthetic = ConvertDecimals(collateralDecimals, syntheticDecimals, this.web3);
+    // Define a set of normalization functions. These Convert a number delimited with given base number of decimals to a
+    // number delimited with a given number of decimals (18). For example, consider normalizeCollateralDecimals. 100 BTC
+    // is 100*10^8. This function would return 100*10^18, thereby converting collateral decimals to 18 decimal places.
+    this.normalizeCollateralDecimals = ConvertDecimals(collateralDecimals, 18, this.web3);
```

Comment:
nit: Not sure if this should be changed here or in another PR, but I don't think `ConvertDecimals` needs to take in web3, it should just be able to import it, right?

Reply:
ye, you right. it does not need it. I'd rather leave this for a separate PR as it will ripple into other changes and this PR is already pretty big.